### PR TITLE
Add JSON marshaling tests

### PR DESCRIPTION
**Pull request by renjithr23**
Moved JSON marshaling test for type Reviewers to appropriate test file. 

Add  JSON marshaling test for type ReviewersRequest

helps - #55

### CODE REVIEW

File: github/pulls_reviewers_test.go [hunk 1]
```diff
@@ -13,6 +13,110 @@ import (
 	"testing"
 )
 
+func TestReviewersRequest_marshall(t *testing.T) {
```
Comment (gmlewis) [inline github/pulls_reviewers_test.go:16]: While you are here, can you please change this from `marshall` to `marshal` ?

File: github/pulls_reviewers_test.go [hunk 2]
```diff
@@ -13,6 +13,110 @@ import (
...
+		],
+		"team_reviewers" : [
+			"t"
+		]
+	}`
+
+	testJSONMarshal(t, u, want)
+}
+
+func TestReviewers_marshall(t *testing.T) {
```
Comment (gmlewis) [inline github/pulls_reviewers_test.go:38]: Same here... can you please change this from `marshall` to `marshal` ?

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @renjithr23 !
Just a couple minor tweaks, please, while you are making this change, if you don't mind.

File: github/pulls_reviewers_test.go (hunk 2 above)
Comment (renjithr23) [reply github/pulls_reviewers_test.go]: Made the change

**Review by gmlewis — APPROVED** [review]
Thank you, @renjithr23 .
In the future, please don't use force push for PRs in this repo, as it makes it difficult for reviewers to see what changed since the last review.  In this case, it's not a big deal.
LGTM.
Merging.

**Comment by renjithr23** [comment]
@gmlewis what is the ideal way for making changes to PRs ? Is it ok to add a new commit , even if it is for a small change to a PR ?

**Comment by gmlewis** [comment]
Hi @renjithr23 - yes, it is totally fine in this repo to push as many commits as you want to a PR... this is because we "squash and merge" each PR so that it ends up being a single commit. You can read more about this in our CONTRIBUTING.md doc.

**Pull request merged**